From 9d8bc305cc919b34c11933a66a31aa11a5abfcbc Mon Sep 17 00:00:00 2001
From: Thomas Richard <thomas.richard@bootlin.com>
Date: Wed, 5 Feb 2025 16:26:11 +0100
Subject: [PATCH] Revert "usb: dwc2: hcd: initialize hsotg->test_mode upon
 USB_PORT_FEAT_TEST"

This reverts commit 10c68f514e6aedf8e91f8905fb924fe12b01765c.
---
 drivers/usb/dwc2/hcd.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/dwc2/hcd.c
+++ b/drivers/usb/dwc2/hcd.c
@@ -3719,7 +3719,6 @@ static int dwc2_hcd_hub_control(struct d
 			hprt0 &= ~HPRT0_TSTCTL_MASK;
 			hprt0 |= (windex >> 8) << HPRT0_TSTCTL_SHIFT;
 			dwc2_writel(hsotg, hprt0, HPRT0);
-			hsotg->test_mode = windex >> 8;
 			break;
 
 		default:
